src/libaktualizr/utilities/events.h

```diff
@@ -97,12 +100,16 @@ class DownloadComplete : public BaseEvent {
   static DownloadComplete fromJson(const std::string& json_str);
 };
 
-/**
- * The UptaneInstall command completed successfully.
- */
+class InstallStarted : public BaseEvent {
+ public:
+  InstallStarted(Uptane::EcuSerial serial_in);
+  Uptane::EcuSerial serial;
+};
+
 class InstallComplete : public BaseEvent {
  public:
-  explicit InstallComplete();
+  InstallComplete(Uptane::EcuSerial serial_in);
+  Uptane::EcuSerial serial;
```

Comment:
We should still have doxygen-style comments for each Event class. Even if they are fairly trivial, we need to have a framework of basic documentation.

Reply:
That's a good point actually. Do we want a better (or at least some) documentation policy? Right now random things get documented whenever it comes to someone's mind.

Reply:
The current requirement, as I understand it, is that anything visible through the libaktualizr API should be documented. That's basically every function in the `Aktualizr` class and every `Event` type that we expect to emit to an API user. Maybe the `Config`. Otherwise, it doesn't matter too much. My current concern is getting this API in a usable state, so anything else is just for us. I'm open to discussion on what is useful.

Reply:
What's the API? Just everything that is in the headers or did we define some outwards facing subset?

Reply:
As far as I'm concerned, it's the aktualizr.h and by extension also event.h.

Reply:
Makes sense, thank you.